homeassistant/components/swiss_public_transport/sensor.py

```diff
@@ -54,7 +54,11 @@ async def async_setup_entry(
         assert unique_id
 
     async_add_entities(
-        [SwissPublicTransportSensor(coordinator, unique_id)],
+        [
+            SwissPublicTransportDepartureSensor(coordinator, unique_id),
+            SwissPublicTransportNextDepartureSensor(coordinator, unique_id),
+            SwissPublicTransportNextOnDepartureSensor(coordinator, unique_id),
```

Comment:
Please checkout sensor entity descriptions

Reply:
@joostlek What do you mean with sensor entity descriptions? Do you mean the naming of the class?

I made some other changes, thus I am not sure if it still applies anymore.

Reply:
I was looking at: https://developers.home-assistant.io/docs/core/entity/sensor

Reply:
Or were you referencing: [_attr_has_entity_name](https://developers.home-assistant.io/docs/core/entity#entity-description)?

Reply:
Nvm, I think I figured it out, kinda. I doubt everything is already perfect, but looking forward to your feedback :)